content/applications/websites/ecommerce/maximizing_revenue/cross_selling.rst

```diff
@@ -1,55 +1,68 @@
-=============================================================
-How to sell accessories and optional products (cross-selling)
-=============================================================
+================================================
+Cross-selling: Accessories and Optional Products
+================================================
 
-You sell computers. Why not stimulating your customers to buy a top-notch
-screen or an extra-warranty? That's the goal of cross-selling functionalities: 
+Any sales process is an opportunity to maximize revenues. For example, with an extra-warranty
```

Comment:
"extra warranty" (no hyphen) or maybe "extended warranty" that is more accurate.